dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/deduplication/PotentialDuplicatesTests.java

```diff
@@ -29,60 +29,59 @@
 
 import static org.hamcrest.Matchers.*;
 
-import java.util.Arrays;
-
 import org.hisp.dhis.dto.ApiResponse;
 import org.hisp.dhis.helpers.JsonObjectBuilder;
 import org.hisp.dhis.helpers.QueryParamsBuilder;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.ValueSource;
 
 /**
  * @author Gintare Vilkelyte <vilkelyte.gintare@gmail.com>
  */
-public class PotentialDuplicatesTests
-    extends PotentialDuplicatesApiTest
+public class PotentialDuplicatesTests extends PotentialDuplicatesApiTest
 {
     @BeforeEach
     public void beforeEach()
     {
         loginActions.loginAsAdmin();
     }
 
-    @ParameterizedTest
-    @ValueSource( strings = { "OPEN", "INVALID", "MERGED" } )
```

Comment:
Is there a reason why you wanted to get rid of the parameters? In my mind it makes the test look a bit cleaner and also runs the test for every status regardless of failures in other statuses. With the change, validation errors in, say OPEN filter, will block other checks from executing.